src/style.css

```diff
@@ -0,0 +1,128 @@
+
+@font-face {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir", sans-serif;
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+}
```

Comment:
you still have the same names
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96265844/1a90e7a2-f476-4839-9ed5-8d1d55ec6dc3)